osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-bundle.xsd

```diff
@@ -254,6 +254,13 @@
       </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
+  <xsd:element name="DeCoupleMbusDeviceByChannelRequest">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="sminst:DeCoupleMbusDeviceByChannelRequestData"></xsd:extension>
```

Comment:
`<xsd:extension base="sminst:DeCoupleMbusDeviceByChannelRequestData"/>` would be more consistent.

Reply:
Removed this and similar inconsistenties from
 - sm-bundle.xsd
 - sm-installation.xsd
 - sm-monitoring.xsd

Reply:
The sm-bundle.xsd has been indented with tabs, convention used across the XSDs is to indent with two spaces.
Avoid tabs wherever they are not required,